src/main/kotlin/com/oyster/dto/Transaction.kt

```
@@ -0,0 +1,6 @@
+package com.oyster.dto
+
+import java.math.BigDecimal
+
+data class Transaction (val fare :BigDecimal, val journey: Journey) {
+}
```

detekt

[warning] The class or object Transaction is empty.

(detekt.empty-blocks.EmptyClassBlock)